XenAdmin/Dialogs/OptionsPages/ConnectionOptionsPage.cs

```diff
@@ -107,6 +115,8 @@ private void build()
                 ProxyUsernameTextBox.Visible = false;
                 ProxyPasswordLabel.Visible = false;
                 ProxyPasswordTextBox.Visible = false;
+                AuthenticationMethodPanel.Visible = false;

```

Comment:
If build() is called only once in the constructor, then it's fine how it is.
We don't normally update the options dialog while it's open so no need to do it here.